lib/puppet/application/cert.rb

```diff
@@ -265,8 +265,10 @@ def main
     end
     begin
       if subcommand == :destroy
+        raise "Refusing to destroy all certs, provide an explicit list of certs to destroy" if hosts == :all
```

Comment:
I think this needs to be wrapped in `_(..)` for string i18n.

Reply:
Originally I thought the string I was copying from wasn't internationalized, but that appears to be because I was only looking at 4.10.x (it's internationalized on master), and didn't think it was really worth it since we've got near-term plans to gut the cert app, and unify its functionality with other cert related apps into a single app.

One odd thing with how the internationalization is being done with the original string is that we're shoving English words in the middle of the i18n string, which doesn't seem very conducive to being correct in the localized string. So...while we _could_ use the same i18n string here as in `Puppet::SSL::CertificateAuthority::Interface`, I think it'd probably be best to inline the words that would have been injected (especially since they'll never change in this context). 

https://github.com/puppetlabs/puppet/blob/3b3f4a81d3f5fa05e31edb7bfc27e0891aa88fd0/lib/puppet/ssl/certificate_authority/interface.rb#L27